src/ngraph/op/topk.cpp

```diff
@@ -76,6 +91,33 @@ void op::TopK::set_k(size_t k)
         op::Constant::create(element::i64, Shape{1}, {k})->output(0));
 }
 
+size_t op::TopK::get_top_k_axis() const
+{
+    auto d = get_top_k_axis_dynamic();
+    NGRAPH_CHECK(d.is_static(),
```

Comment:
does this require top_k_axis to be a constant node? Otherwise it is not supported yet?

Reply:
`get_top_k_axis()` will not work if the value of `k` is unknown. This is generally how things work with these accessors for "formerly static attributes".

Reply:
In general, this won't cause issues for the moment---code that expects top_k_axis to be statically known isn't going to be invoked until after graph specialization.